### DISCUSSION TITLE
PermissionError: [Errno 13] Permission denied: 'C:\\.webui_secret_key'

### DISCUSSION THREAD

**Discussion by zhaowei0315** [open-webui/open-webui#2625] [General]
# Bug Report
  PermissionError: [Errno 13] Permission denied: 'C:\\.webui_secret_key'
## Description
  Could relocate file .webui_secret_key to home directory (e.g. C:\Users\xxxx) instead of root folder of C for windows OS?

**Comment by justinh-rahb** [comment]
Don't run from the root of drive C:\

**Comment by NikeLaosClericus** [comment]
cd ~
